src/qtism/data/storage/xml/marshalling/Marshaller.php

```diff
@@ -325,35 +329,43 @@ public function __call($method, $args)
             case 'marshall':
                 $component = $args[0];
                 $this->checkMarshallerImplementation($component);
+
                 return $this->marshall($component);
 
             case 'unmarshall':
                 $this->checkUnmarshallerImplementation($args[0]);
+
                 return $this->unmarshall(...$args);
 
             default:
                 throw new RuntimeException("Unknown method Marshaller::'${method}'.");
         }
     }
 
-    protected function checkMarshallerImplementation($component)
+    protected function checkMarshallerImplementation($component): void
     {
-        if (!$component instanceof QtiComponent || ($this->getExpectedQtiClassName() !== '' && $component->getQtiClassName() !== $this->getExpectedQtiClassName())) {
+        if (!$component instanceof QtiComponent || ($this->getExpectedQtiClassName(
+                ) !== '' && $component->getQtiClassName() !== $this->getExpectedQtiClassName())) {
             $componentName = $this->getComponentName($component);
-            throw new RuntimeException("No marshaller implementation found while marshalling component '${componentName}'.");
+
+            throw new RuntimeException(
+                "No marshaller implementation found while marshalling component '${componentName}'."
+            );
         }
     }
 
-    protected function checkUnmarshallerImplementation($element)
+    protected function checkUnmarshallerImplementation($element): void
     {
-        if (!$element instanceof DOMElement || ($this->getExpectedQtiClassName() !== '' && $element->localName !== $this->getExpectedQtiClassName())) {
+        if (!$element instanceof DOMElement || ($this->getExpectedQtiClassName(
+                ) !== '' && $element->localName !== $this->getExpectedQtiClassName())) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place